Apps.md

```diff
@@ -0,0 +1,31 @@
+# BTCPay Server Apps
+The primary purpose of BTCPay Server is to remove dependencies on trusted third-parties. The Apps are built in applications that obsolete central-authorities and allow users an easy way to extend the [use case](UseCase.md) of the software. Users can self-host all sort of customizable applications that work out of the box. 
+
+To create an app, go to Apps > Create a new app. Apps are store-dependant, which means that each app needs to be connected to a store.
+
+## Point of Sale App
+The web-based PoS app allows users with brick and mortar stores to readily accept cryptocurrencies without fees or a third-party, directly to their wallet. 
+
+![Invoice exporting in BTCPay](img/BTCPayPointOfSale1.jpg)
```

Comment:
Link isn't working. Is export supposed to be a feature? or just a feature of PoS apps?

Reply:
sorry i didn't see it was a jpg